manifests/kots-config.yaml

```diff
@@ -349,10 +349,12 @@ spec:
               title: |-
                 Custom
         - name: accessToCartoModeK8sNotFullySupported
+          hidden: true
```

Comment:
Why is this one hidden? 🤔 We are rendering it just when needed no?

Reply:
Oh got it, makes sense to me! Thanks!

Reply:
Yes, but I think that it is a bit strange to show only the "Custom" option if it is the only possibility to select 🤔